libr/main/radare2.c

```diff
@@ -1328,8 +1328,14 @@ R_API int r_main_radare2(int argc, char **argv) {
 		r_flag_space_set (r.flags, NULL);
 		/* load <file>.r2 */
 		{
-			char f[128];
+			char f[256];
 			snprintf (f, sizeof (f), "%s.r2", pfile);
```

Comment:
we should use r_str_newf () instead here, having a 256 char limit here is not a good idea